map2loop/thickness_calculator.py

```diff
@@ -394,6 +418,8 @@ def __init__(self):
         self.thickness_calculator_label = "StructuralPoint"
         self.line_length = 10000
         self.strike_allowance = 30
+        self.max_line_length = None
+        self.lines = []
```

Comment:
Don't initialise lines as a list and then change it to a geopandas dataframe, make it either None or an empty list

Reply:
https://github.com/Loop3D/map2loop/pull/161/commits/6a43ee9de156823a39e5db6b097e5e8be7245904